docs/user-documentation/intro-to-ld-for-claw.md

```diff
@@ -0,0 +1,68 @@
+
+# Basics of Linked Data
+This section seeks to give the reader a foundational understanding of what linked data is, why it is useful, and a very superficial understanding of how it works.
+- [Tim Berners-Lee’s description of Linked Data](https://www.w3.org/DesignIssues/LinkedData.html)
+- [Manu Sporny's "What is Linked Data?" YouTube Video](https://www.youtube.com/watch?v=4x_xzT5eF5Q)
+- [Wikipedia article on Linked Data](https://en.wikipedia.org/wiki/Linked_data)
+- [Wikipedia article on Semantic Web](https://en.wikipedia.org/wiki/Semantic_Web)
+- [Wikipedia article on URIs](https://en.wikipedia.org/wiki/Uniform_Resource_Identifier)
+- [Wikipedia article on the W3C](https://en.wikipedia.org/wiki/World_Wide_Web_Consortium)
+- [W3C’s description of Linked Data](https://www.w3.org/standards/semanticweb/data)
+- [W3C’s Linked Data Glossary](https://www.w3.org/TR/ld-glossary/)
+- [W3C’s Architecture of the World Wide Web](https://www.w3.org/TR/webarch/)
+
+# Understanding RDF
+This section is all about RDF, the Resource Description Framework, which defines the way linked data is structured.
+- [Wikipedia article on RDF](https://en.wikipedia.org/wiki/Resource_Description_Framework)
+- [D-Lib’s Intro to RDF](http://www.dlib.org/dlib/may98/miller/05miller.html)
+- [W3C’s RDF 1.1 Primer](https://www.w3.org/TR/rdf11-primer/)
+- [W3C’s RDF 1.1 Concepts](https://www.w3.org/TR/rdf11-concepts/)
+
+# Querying Linked Data with SPARQL
+This section takes a look at SPARQL, the query language that allows you to ask linked data very specific questions. The queryable nature of linked data is one of the things that makes it so special. Try some SPARQL queries on DBpedia's endpoint to get some hands-on experience.
+- [Wikipedia article on SPARQL](https://en.wikipedia.org/wiki/SPARQL)
+- [W3C’s SPARQL 1.1 Overview](https://www.w3.org/TR/sparql11-overview/)
+- [W3C’s SPARQL 1.1 Query Language](https://www.w3.org/TR/sparql11-query/)
+- [DBpedia's SPARQL Endpoint](https://dbpedia.org/sparql)
+
+# RDF Serialization Formats
+RDF data can be translated into many different formats. RDF/XML is the original way that RDF data was shared, but there are much more human-friendly serialization formats like Turtle which is great for beginners. JSON-LD is the easiest format for applications to use, and is the serialization format that CLAW uses internally.
+- [Wikipedia article on Serialization](https://en.wikipedia.org/wiki/Serialization)
+- [W3C’s RDF/XML Syntax Specification](https://www.w3.org/TR/REC-rdf-syntax/)
+- [W3C’s RDF 1.1 Turtle](https://www.w3.org/TR/turtle/)
+- [W3C’s JSON-LD 1.0](https://www.w3.org/TR/json-ld/)
```

Comment:
http://json-ld.org
and
http://json-ld.org/playground/
are great JSON-LD resources